src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/TeamQueries.cs

```diff
@@ -142,12 +142,14 @@ public Team Execute(FindByNameCriteria criteria)
 
         private static Team GetTeamMapping(TeamEntity t)
         {
+            var roster = t.Players?.Select(p => new PlayerId(p.Id)) ?? new List<PlayerId> { new PlayerId(t.CaptainId) };
```

Comment:
Why do you need this?

Reply:
I changed this, problem was in nullrefexception